docs/guide/execute-transaction.md

```diff
@@ -15,6 +14,43 @@ Note: For each parameter `type` and `sign` attributes are mandatory.
 
 Depending on the transaction `type`, other attributes will specify a signer of the transaction. For example, for `TransactionType.TransferAlgo` the `fromAccount` must be a full account (with secret key) and will sign the transaction.
 
+`deployer.executeTx` returns list of `TxnReceipt`, which extends `ConfirmedTxInfo`.
+
+```js
+const receipts = deployer.executeTx([txn0, txn1]);
+console.log("txn0 information: ", receipts[0]);
+console.log("txn1 information: ", receipts[2]);
+```
+
+Below we demonstrate how to perform application call using `ExecParam` and `WebMode`:
+
+```js
+appCall = () => {
+		const webMode: WebMode = new WebMode(AlgoSigner, networkType);
+		const tx: types.ExecParams[] = [
+			{
+				type: types.TransactionType.CallApp,
+				sign: types.SignType.SecretKey,
+				fromAccount: {
+					addr: addr,
+					sk: new Uint8Array(0),
+				},
+				appID: 100,
+				payFlags: {},
+			},
+		];
+		webMode.executeTx(tx);
+	};
+```
+
+We can also use `SignedTransaction` and `executeTx` in runtime:
```

Comment:
we could add link to an example from Algorand docs with the signedTransaction. How it is created using `algosdk`. Or just add it to our example

Reply:
We already have signed transaction example at the end of this doc.